baksnapper.sh

```diff
@@ -730,10 +726,14 @@ esac
 if [[ ${p_prune-0} == 1 ]]
 then
     $receiver remove-snapshots "$dest_root" "${only_in_dest[@]}"
-    if [[ $p_link -eq 1 ]]
-    then
-        $receiver link-latest "$dest_root"
-    fi
 fi
 
+if [[ $p_link -eq 1 ]]
+then
+    $receiver link-latest "$dest_root"
+fi
+
+echo "--- Statistics ---"
 cat "${p_summary}"
+echo "--- Statistics ---"
+rm -rf "${p_temp_dir}"
```

Comment:
This is out of scope for this PR.

And also the `rm -rf …` should be handled by the `cleanup` function.